src/components/playerOverview/MatchPlacementBarChart.tsx

```diff
@@ -20,8 +21,13 @@ export const MatchPlacementBarChart = React.memo(function MatchPlacementBarChart
     }
 
     const matchPlacementDictionary: { [rank: string]: number } = {};
+    const validMatches = filterMatchesByPlayerCount(matches, NUMBER_OF_PLAYERS_FOR_VALID_MATCH);
 
-    for (const match of matches) {
+    if (validMatches === undefined) {
+        return <Text textAlign={"center"}>No valid matches to display.</Text>
```

Comment:
nit: Can can probably drop the "to display." part of this text and just say "No valid matches".

Reply:
Done in both playerOverview and commanderOverview.